lib/kinesis_client.rb

```diff
@@ -41,24 +41,25 @@ def <<(json_message)
     send(@shovel_method, json_message)
   end
 
-  def push_record(json_message)
-    record = convert_to_record(json_message)
-    record[:stream_name] = @stream_name
-
-    @client.put_record(record)
-
-    return_hash = {}
-
-    if resp.successful?
-      return_hash["code"] = "200"
-      return_hash["message"] = json_message, resp
-      $logger.info("Message sent to #{config[:stream_name]} #{json_message}, #{resp}") if $logger
-    else
-      $logger.error("message" => "FAILED to send message to #{@stream_name} #{json_message}, #{resp}.") if $logger
-      raise(NYPLError.new(json_message, resp))
-    end
-    return_hash
-  end
+#This method is broken
+  # def push_record(json_message)
```

Comment:
Let's not comment this out just yet. I want to try to find out if any other applications are using this method.